src/openms/source/ANALYSIS/OPENSWATH/MRMFeatureFilter.cpp

```diff
@@ -166,7 +166,9 @@ namespace OpenMS
               String component_name2 = (String)features[feature_it].getSubordinates()[sub_it2].getMetaValue("native_id"); 
 
               // find the ion ratio pair
-              if (filter_criteria.component_group_qcs[cg_qc_it].ion_ratio_pair_name_1 == component_name
+              if (filter_criteria.component_group_qcs[cg_qc_it].ion_ratio_pair_name_1 != ""
+                && filter_criteria.component_group_qcs[cg_qc_it].ion_ratio_pair_name_1 != ""
```

Comment:
typo: _1 should be _2 in the second line I guess

Reply:
Typo indeed.  I will correct that now.